### PR TITLE
[CLEANUP] Improve the type annotations for some `CssInliner` fields

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: src/CssInliner.php [hunk 1]
```diff
@@ -93,22 +93,22 @@ class CssInliner extends AbstractHtmlProcessor
     ];
 
     /**
-     * @var CssSelectorConverter
+     * @var CssSelectorConverter|null
```
Comment (JakeQZ) [inline src/CssInliner.php:96]: Is `@var ?CssSelectorConverter` possible/preferable?

File: psalm.baseline.xml [hunk 2]
```diff
@@ -51,6 +49,10 @@
       <code>string[]</code>
       <code>string[][][]</code>
     </MixedInferredReturnType>
+    <MixedPropertyTypeCoercion occurrences="2">
+      <code>$this-&gt;styleAttributesForNodes</code>
+      <code>$this-&gt;visitedNodes</code>
+    </MixedPropertyTypeCoercion>
```
Comment (JakeQZ) [inline psalm.baseline.xml:55]: I don't understand why we now have these errors but didn't before.  Though I note the overall error count has gone down :)

**Review by JakeQZ — COMMENTED** [review]
Looks OK but I have a couple of questions.

File: src/CssInliner.php (hunk 1 above)
Comment (oliverklee) [reply src/CssInliner.php:96]: Yes, changed.

File: psalm.baseline.xml (hunk 2 above)
Comment (oliverklee) [reply psalm.baseline.xml:55]: Now that the field types are more specific, Psalm can recognize cases where the code (or - more probable - the annotations for that code) does not make sure (yet) that these more specific type restrictions are adhered to.

Comment (JakeQZ) [reply psalm.baseline.xml:55]: OK.

**Review by JakeQZ — APPROVED** [review]

**Pull request merged**